js/samples/03.ai-concepts/f.chatModeration/infra/azure.parameters.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "resourceBaseName": {
+            "value": "bot${{RESOURCE_SUFFIX}}"
+        },
+        "botAadAppClientId": {
+            "value": "${{BOT_ID}}"
+        },
+        "botAadAppClientSecret": {
+            "value": "${{SECRET_BOT_PASSWORD}}"
+        },
+        "webAppSKU": {
+            "value": "B1"
+        },
+        "botDisplayName": {
+            "value": "ChatBotModeration"
+        },
+        "openAIKey": {
```

Comment:
Do we need to include AOAI key here as well?

Reply:
Not in azure.parameters.json - we can't comment it out (causes errors) so it's in the readme to add it manually.